doc/intro/compiling.rst

```diff
@@ -207,3 +207,9 @@ In a different terminal, run the module code for each module that you want to co
 
     $ source setup.sh -f Debug # Setup your terminal
     🤖 D $ basicJointPosition  # Run the compiled module code
+
+If you are interested in controlling the simulation via Python instead of C++,
+you can follow example `examples/simulator/pythonInterface`:
+
+.. code-block:: sh
```

Comment:
I would add the setup command here as well (I see that the robot icon is there, but I didn't realize it at first)

Reply:
I would leave only the command there.